### ISSUE TITLE
Limit tabs list to 5 (and a "view more" collpase)

### ISSUE THREAD

**Issue by jerembdn** [onRuntime/kartrak#40]
- [ ] Limit tabs list to 5
- [ ] Add ability to expand and see the full list
- [ ] Up the active tab as 1st item (but mention the actual position of the tab ?)

**Comment by jerembdn** [comment]
@AntoineKM as an active contributor this could be interesting !!

**Comment by AntoineKM** [comment]
take a look at my `antoinekm/range` branch

**Comment by jerembdn** [comment]
> take a look at my `antoinekm/range` branch

I see nothing.. ?

**Comment by AntoineKM** [comment]
don't you have an active tab indicator?

**Comment by AntoineKM** [comment]
the tabs displayed are in order and are those of the active window. i rly don't know if changing this order would improve the UX

**Comment by jerembdn** [comment]
> don't you have an active tab indicator?

Yes but this issue is about up the active tab on top of the list

**Comment by jerembdn** [comment]
> the tabs displayed are in order and are those of the active window. i rly don't know if changing this order would improve the UX

I think users would be less lost when the list will be collpased ?
